DependencyInjection/Compiler/EntityListenerPass.php

```diff
@@ -48,8 +48,41 @@ public function process(ContainerBuilder $container)
                     continue;
                 }
 
+                if (isset($attributes['entity']) && isset($attributes['type'])) {
+                    $this->attachToListener($container, $name, $id, $attributes);
+                }
+
                 $container->getDefinition($resolver)->addMethodCall('register', array(new Reference($id)));
             }
         }
     }
+
+    private function attachToListener(ContainerBuilder $container, $name, $id, array $attributes)
+    {
+        $listenerId   = sprintf('doctrine.orm.%s_listeners.attach_entity_listeners', $name);
+        if ($container->hasAlias($listenerId)) {
+            $listenerId = (string)$container->getAlias($listenerId);
+        }
+
+        if (!$container->hasDefinition($listenerId)) {
+            return;
+        }
+
+        $serviceDef = $container->getDefinition($id);
+
+        $args = array(
+            $serviceDef->getClass(),
+            $attributes['entity'],
```

Comment:
this should be

```php
            $attributes['entity'],
            $serviceDef->getClass(),
```

See https://github.com/doctrine/doctrine2/blob/master/docs/en/changelog/migration_2_5.rst#events-add-api-to-programatically-add-event-listeners-to-entity